### PR TITLE
{toolchain} fosscuda/2018b

### PR DESCRIPTION
**Pull request by akesandgren**
(created using `eb --new-pr`)
Based on foss/2018b and CUDA 9.2.88, includes HPL

### CODE REVIEW

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
b-cn1522.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) Gold 6132 CPU @ 2.60GHz, Python 2.7.12
See https://gist.github.com/9b1b9fb8621641907809ea6e02ffad43 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node2577.golett.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/04bdb5bb6b7b5075bb8341083b1cda80 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node2082.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/59b397fc3fac0402db75e4ac235d9cd0 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 8 out of 8 (8 easyconfigs in this PR)
node3159.skitty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/cee7b7225092d91d2e9525f61d258616 for a full test report.

File: easybuild/easyconfigs/h/HPL/HPL-2.2-fosscuda-2018b.eb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+description = """HPL is a software package that solves a (random) dense linear system in double precision (64 bits)
+ arithmetic on distributed-memory computers. It can thus be regarded as a portable as well as freely available
+ implementation of the High Performance Computing Linpack Benchmark."""
+
+toolchain = {'name': 'fosscuda', 'version': '2018b'}
+toolchainopts = {'usempi': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://www.netlib.org/benchmark/%(namelower)s']
+checksums = ['ac7534163a09e21a5fa763e4e16dfc119bc84043f6e6a807aba666518f8df440']
```
Comment (boegel) [inline easybuild/easyconfigs/h/HPL/HPL-2.2-fosscuda-2018b.eb]: missing checksum for patch, cfr. https://github.com/easybuilders/easybuild-easyconfigs/pull/6424/files#diff-21f46ca04de12d4b01d62a39ce2a0686R14

**Comment by boegel** [comment]
Going in, thanks @akesandgren!

**Pull request merged**